detection_rules/packaging.py

```diff
@@ -470,6 +477,24 @@ def create_bulk_index_body(self) -> Tuple[Ndjson, Ndjson]:
 
         return bulk_upload_docs, importable_rules_docs
 
+    @staticmethod
+    def add_historical_rules(historical_rules: Dict[str, dict], manifest_version: str) -> list:
+        """Adds historical rules to existing build package."""
+        rules_dir = CURRENT_RELEASE_PATH / 'fleet' / manifest_version / 'kibana' / 'security_rule'
+        for historical_rule_id, historical_rule_contents in historical_rules.items():
+            rule_id = historical_rule_contents["attributes"]["rule_id"]
+            historical_rule_version = historical_rule_contents['attributes']['version']
+            current_rule_path = list(rules_dir.glob(f"{rule_id}*.json"))
+            if not current_rule_path:
+                continue
+            current_rule_path = current_rule_path[0]
+            current_rule_json = json.load(current_rule_path.open(encoding="UTF-8"))
+            current_rule_version = current_rule_json['attributes']['version']
+            if historical_rule_version != current_rule_version:
```

Comment:
nit: might be good to add a single doctoring to summarize the logic.

Reply:
Will do.